include/llvm/CodeGen/AsmPrinter.h

```diff
@@ -112,6 +112,9 @@ class AsmPrinter : public MachineFunctionPass {
   typedef std::pair<const GlobalVariable *, unsigned> GOTEquivUsePair;
   MapVector<const MCSymbol *, GOTEquivUsePair> GlobalGOTEquivs;
 
+  /// Sections that need to be referenced in the PGLT
+  std::vector<const MCSection*> PGLT;
+
```

Comment:
Sure, could push nullptr as index. I guess that makes more sense than 1-based indexing (as you commented above).